.github/workflows/update-fog.yml

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+# This workflow periodically calls the fog-update-bot action to update
+# the list of FOG metrics.yaml and ping.yaml files from its metrics_index.py
+
+
+name: "Update FOG"
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  schedule:
+    # 04:00 UTC - every morning
```

Comment:
In what way are we guaranteed that this is interpreted wrt UTC?

Reply:
GitHub Actions docs [say so](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) (hm, might want to shift that by a bit just to not run into the called out "high load" periods.